packages/sources/css/src/components/structure/list/src/index.css

```diff
@@ -18,6 +18,19 @@
   padding: 0;
 }
 
+.vtmn-list a.vtmn-list__link {
+  display: flex;
+  inline-size: 100%;
+  text-decoration: inherit;
+  color: inherit;
+}
+
+.vtmn-list li > a.vtmn-list__link[aria-disabled='true'] {
```

Comment:
Not appropriate to use `aria-disabled` maybe using `disabled` is more reliable

Reply:
Not possible -> https://www.scottohara.me/blog/2021/05/28/disabled-links.html
The best way here is to used `aria-disabled` on the link

Reply:
Well, can we create a status class instead ? Like `is-disabled`?

Reply:
@thibault-mahe what's your point of view ?

Reply:
`is-disabled` would be for the style, but `aria-disabled` is the real thing for assistive technology. Why is it not appropriate ?

Reply:
because we base appearence of the button on the assertive attribute while styles is for everyone.
For me, aria is a complement to help screen readers to better understand our interfaces but not to use it as a CSS Selector.

As we discuss with Antoine, for this particular case, it's a good solution, but we missing some rules/good practices about the fact to use aria attributes as css selectors.

Reply:
Yes you right, I didn't fully understand the context, my bad.
The CSS must not have any dependency with the DOM structure, I totally agree.
Currently our css is already very dependent on the DOM so this is something we will work on for v1